### DISCUSSION TITLE
"Add a download button for the project page" Credit

### DISCUSSION THREAD

**Discussion by LilSefGaming** [ScratchAddons/ScratchAddons#7361] [Ideas]
I made that suggestion maybe two years ago! I should have gotten credit for that! I never got credited though!
Please credit me! Proof! #4309

**Comment by Samq64** [comment]
Usually the person who codes the addon is the main one credited.

**Reply by BroJac5246** [reply]
You can get added to the contributors repo for giving ideas, so it could be the same for getting credited on addons.

**Comment by DNin01** [comment]
If you want, you can refer to https://github.com/ScratchAddons/contributors/issues/250 to learn how to get onto our contributor credits on our website, which can be opened by clicking "Credits" in the SA settings page.
